pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -345,17 +327,25 @@ Answer.Date:
     - $ref: '#/Answer'
     - type: object
       required:
+        - type
         - value
       properties:
+        type:
+          type: string
+          default: DATE
```

Comment:
does it make sense to have a type and a default when there is only a possible option? Could this be written something like this  `{...properties: { type : DATE}, {...properties: { type : COMPOSITE}`?

Reply:
No. First off, the type field only allows raw JSON types (so, array, int, number, string, and object). Second, these values are used by the client to determine the type of the object being returned. It's a form of polymorphism which works in tandem with the discriminator properties you'll also see in the spec.

Reply:
I see it looks good on the docs
![image](https://user-images.githubusercontent.com/98110251/152033961-db3cf487-7501-431d-bae5-d46d609a6bc0.png)
That double type on code got my attention